build.sbt

```diff
@@ -368,7 +401,7 @@ lazy val publishSettings =
 
 lazy val mimaSettings = Seq(
   mimaPreviousArtifacts := {
-    val unpublishedModules = Set[String]()
+    val unpublishedModules = Set[String]("ciris-iron")
```

Comment:
Not sure what this code is supposed to do. As far as I can tell, this is the only difference with other modules. Maybe it is the cause of your dependency problem?

Reply:
This is not part of my branch, i believe this was part of a recent commit by  [vlovgr](https://github.com/vlovgr)  https://github.com/vlovgr/ciris/pull/692/commits/55a3c98705e6efe2600246eb0f0738626245ce36

Reply:
We add modules here when there's no published version yet, to exclude them from mima checks.

Reply:
OK that makes sense and why i was in a vicious circle.